tfhub_dev/assets/rishit-dagli/rishit-dagli.md

```diff
@@ -0,0 +1,13 @@
+# Publisher rishit-dagli
+Rishit Dagli.
+
+[![Icon URL]](https://avatars3.githubusercontent.com/u/39672672?s=460&u=315923c86d9e20cca52085af7eb3190c5bb263e7&v=4)
+
+## Details
+Rishit Dagli ([@rishit_dagli](https://twitter.com/rishit_dagli)).
+10 th Grade Student.
+TedX, TedEd Speaker.
+
+**GitHub:** [Rishit-dagli](https://github.com/sayakpaul)\
```

Comment:
The link is probably incorrect. Should it be https://github.com/Rishit-dagli ?

Reply:
This was my first time contributing so as a reference to the directory structure I took inspiration from @sayakpaul models, seems like I forgot to edit this part. Thanks for pointing it out.